app/src/main/java/com/poupa/vinylmusicplayer/service/MediaButtonIntentReceiver.java

```diff
@@ -190,7 +188,7 @@ private static void acquireWakeLockAndSendMessage(Context context, Message msg,
         if (mWakeLock == null) {
             Context appContext = context.getApplicationContext();
             PowerManager pm = (PowerManager) appContext.getSystemService(Context.POWER_SERVICE);
-            mWakeLock = pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, "Vinyl Music Player headset button");
+            mWakeLock = pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, ":Vinyl Music Player headset button");
```

Comment:
Yes. Lint states: `Tag name should use a unique prefix followed by a colon (found Vinyl Music Player headset button). For instance myapp:mywakelocktag. This will help with debugging.`

Reply:
okay thanks. I have near zero understanding/experience on Android messaging, so I would refrain from commenting further.